tests/integration/targets/identitystore/tasks/main.yml

```diff
@@ -0,0 +1,16 @@
+- name: Run identitystore tests
+  module_defaults:
+    community.aws.identitystore_user:
+      access_key: "{{ aws_access_key }}"
+      secret_key: "{{ aws_secret_key }}"
+      session_token: "{{ security_token | default(omit) }}"
+      region: "{{ aws_region }}"
```

Comment:
@vonschultz Thank you for this contribution. Since there isn’t currently a module available to retrieve the Identity Store ID, I believe it would be better to add a task that uses the AWS CLI to list all instances and then extract the identity_store_id from the output of that task. I'm not familiar with this service, but, what do you think about this suggestion?